workspaces/orchestrator/docs/Contributors.md

```diff
@@ -0,0 +1,70 @@
+## How to use the Git Hub identity provider
+
+The [app config](../app-config.yaml) and the [App.tsx](../packages/app/src/App.tsx) files contain sufficient configuration to use the Git Hub identity provider.
+
+Follow these steps to login to backstage using your github account:
+
+### Create a Github OAuth App
+
+Go to your Github account -> Setting -> Developer settings -> OAuth Apps -> New OAuth App.
+Enter a name, enter http://localhost:3000 to Homepage URL and http://localhost:7007/api/auth/github/handler/frame to Authorization callback URL and click on Register applicatdion.
+
+### Update backstage user entity
+
+Update [users.yaml](../users.yaml) to match you Github account. Put your github username in the metadata.name field, and you github email in spec.profile.email field.
```

Comment:
s/Github/GitHub

Reply:
Perhaps a snippet will be a simpler option of a User entity (just a thought)

Reply:
you have it in ../users.yaml